templates/bitcoin-sample.conf

```diff
@@ -19,8 +19,9 @@ maxconnections=40
 maxuploadtarget=1000
 
 # zmq
-zmqpubrawblock=tcp://0.0.0.0:28332
-zmqpubrawtx=tcp://0.0.0.0:28333
+zmqpubrawblock=tcp://0.0.0.0:<port>
+zmqpubrawtx=tcp://0.0.0.0:<port>
+zmqpubhashblock=tcp://0.0.0.0:<port>
```

Comment:
It's fine to expose these options to apps but ideally it would be done in a separate PR.

Reply:
Gonna let this here for testing purpose, remove it before merge and make another PR. Is that okay for you?

Reply:
Yes, perfect!

Reply:
See #468.